core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.controller.ts

```diff
@@ -103,15 +131,23 @@ angular.module('oppia').directive('topicsAndSkillsDashboardPage', [
                   EVENT_TYPE_SKILL_CREATION_ENABLED, ctrl.userCanCreateSkill);
                 ctrl.userCanDeleteTopic = response.can_delete_topic;
                 ctrl.userCanDeleteSkill = response.can_delete_skill;
-                $rootScope.$apply();
```

Comment:
This is no longer required? Why?

Reply:
It is, I haven't removed it, just moved this to the bottom part.